pkg/controllers/user/ingress/ingress.go

```diff
@@ -123,26 +135,23 @@ func (c *Controller) sync(key string, obj *v1beta1.Ingress) error {
 				return err
 			}
 		} else {
-			ingressManaged := false
-			for _, o := range existing.OwnerReferences {
-				if o.UID == obj.UID {
-					ingressManaged = true
-					break
-				}
+			// TODO - fix so the update is done only when workload ids are changed
+			delete(ingressServices, serviceName)
```

Comment:
we should delete after update was successful. This way we can save on avoiding delete operation if update failed.